app/controllers/admin/employees_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Admin
+  class EmployeesController < AdminController
+    def index
+      render :index, locals: { employees: Employee.includes(:orders).all }
+    end
+
+    def kudos_for_all
+      render :add_kudos_for_all, locals: { employees: Employee }
+    end
+
+    def add_kudos_for_all
+      Employee.all.each do |employee|
+        employee.number_of_available_kudos += add_kudos
+        unless employee.save!
+          redirect_to admin_employee_kudos_for_all_path(Employee),
+                      notice: 'There was an error. Please try again.'
+        end
+      end
+      redirect_to admin_employees_path, notice: 'Number of available Kudos was successfully added to Employees.'
+    end
+
+    private
+
+    def add_kudos
+      params[:add_kudos].to_i
```

Comment:
I added a condition in the form that the value must be in the range 1..20 otherwise the message pops up too small or too large number
 `<%= form.number_field :add_kudos, in: 1..20, value: 10 %>`

Reply:
ok, but what form is really doing? it is passing a parameter(s) with a post method - what if I will send such request not by form?